src/util/strencodings.h

```diff
@@ -301,6 +301,39 @@ bool ConvertBits(O outfn, It it, It end, I infn = {}) {
     return true;
 }
 
+/** Convert from a set of power-of-2 number base to another set. */
+template<bool pad, typename O, typename It, typename I = IntIdentity>
+bool ConvertBitsVariable(std::vector<int> frombitsvec, std::vector<int> tobitsvec, O outfn, It it, It end, I infn = {}) {
+    if (frombitsvec.size() == 0 || tobitsvec.size() == 0) return false;
```

Comment:
In a3908f31e4dc48f87752791e5f04bb3c6006c168:

Why return false here?

Reply:
Removed, Thanks!